include/boost/url/impl/url_base.hpp

```diff
@@ -53,8 +53,34 @@ class url_base::segments_type::iterator
     using value_type =
         segments_type::value_type;
 
+    /** The pointer type of segments_type::iterator.
+
+        Defined as:
+
+        @code
+        using pointer = value_type const*;
+        @endcode
```

Comment:
There's no need for this "Define as", the doc toolchain automatically repeats the declaration.

Reply:
Example: 
![image](https://user-images.githubusercontent.com/1503976/105881284-2805f680-5fb9-11eb-9741-a095b7d35f1d.png)